StockInformation/old_support_extract.py

```diff
@@ -152,9 +149,9 @@ def create_resistance_report(ticker,current_price,previous_close):
         price = " "
 
     all_resistances.clear()
-    return words, break_through, past_time_interval, next_time_interval, resistance, next_resistance
+    return words, break_through, past_time_interval, next_time_interval, resistance, next_resistance, price
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears you are unpacking more than three variables while using a function that returns more than one variable. Using a large number of return values is prone to errors. We recommend that you return a small class or `namedtuple` instance instead.

[Learn more](https://docs.python.org/3/library/typing.html#typing.NamedTuple)